html/rendering/replaced-elements/attributes-for-embedded-content-and-images/content-aspect-ratio.html

```diff
@@ -0,0 +1,24 @@
+<!doctype html>
+<title>div with content style's width and height attributes are not used to infer aspect-ratio</title>
```

Comment:
It'd be helpful to add a `<link rel="help">` pointing to the webkit bug / comment or something so that it doesn't feel out of place.

Reply:
Done. Thanks